### DISCUSSION TITLE
Support of TMS and PM Tiles

### DISCUSSION THREAD

**Discussion by IGNF-Xavier** [geonetwork/geonetwork-ui#928] [Ideas]
I suggest that geonetwork-ui supports TMS (OGC) and PMTiles in all of these components: 
- as a link in the API section #922 
- as data viewable in the mapviewer with management of available styles #732 
- as a base map in the mapviewer with a style proposed by default by the one deploying the catalog #920 
- as an addable resource in the mapviewer (i am not sure if it is in this ticket #756) 

I should have propose this discussion before adding tickets. Sorry.
This is part of a context where vector tiles are increasingly used

For the community, can we have an estimation of the cost of this feature (if recognized as useful by the community)

**Comment by jahow** [comment]
Related to https://github.com/geonetwork/geonetwork-ui/issues/726 as well
